### ISSUE TITLE
Tagaini Jisho_Vocabulary-> Parts of Speech is not displayed all the list

### ISSUE THREAD

**Issue by sandvasam** [Gnurou/tagainijisho#232]
![Tagaini Error](https://user-images.githubusercontent.com/63537976/79065195-0bb7f800-7ccc-11ea-8233-4a38d3da0701.jpg)
[](url)

**Comment by Gnurou** [comment]
Yeah the list of part of speeches has a bit grown out of control. I am looking at adding some hierarchy to it.

**Comment by mbruckschwaiger** [comment]
The grouping of POS could make it possible to search for all GODAN verbs for instance if GODAN verbs are grouped in a hierarchy. Right now selecting more than one 'Godan verb with ... ending' combines these selection with AND. An OR combination could be realized with a hierarchy. A search for all GODAN verbs would be great. Generally speaking, an AND combination of different POS would be great of course. The Godan verbs are just an example.
